src/components/PreJoinScreens/DeviceSelectionScreen/DeviceSelectionScreen.tsx

```diff
@@ -71,6 +72,21 @@ export default function DeviceSelectionScreen({ name, roomName, setStep }: Devic
     });
   };
 
+  if (isFetching || isConnecting) {
+    return (
+      <Grid container justify="center" spacing={2} alignItems="center" direction="column" style={{ height: '100%' }}>
```

Comment:
Nit - I'm finding that `spacing={2}` is adding some styles that are making it so the loading wheel isn't perfectly centered. These are the styles that are added:

```
MuiGrid-spacing-xs-2 {
    width: calc(100% + 16px);
    margin: -8px;
}
```

Let's remove it so that it is perfectly centered.
```suggestion
      <Grid container justify="center" alignItems="center" direction="column" style={{ height: '100%' }}>
```